src/server/scripts/EasternKingdoms/zone_alterac_mountains.cpp

```diff
@@ -31,7 +31,7 @@ EndContentData */
 #include "ScriptMgr.h"
 #include "ScriptedCreature.h"
 
-/*void AddSC_alterac_mountains()
+void AddSC_alterac_mountains()
 {
-    Script* newscript;
-}*/
+
+}
```

Comment:
not sure about this one i would maybe exclude it from build there is no reason to include stuff and increase compile time for a place holder.

Reply:
I was thinking about removing it completly (can be added later) but there were a little more empty ones like this. Can check and remove all of them.